### ISSUE TITLE
  subscript out of bounds

### ISSUE THREAD

**Issue by afshinmoradi** [plevritis-lab/CELESTA#13]
Dear,

I am getting this error. is there any idea how to fix it

Error in `[<-`(`*tmp*`, unassigned_cells, cell_type_num[i], value = GetScore(activation_prob_to_use,  : 
  subscript out of bounds

Best regards,

afshin

**Comment by weiruo16** [comment]
Hi, we have fixed the issue with the unassigned cells. Please try it again.

**Comment by Elena983** [comment]
Hi
I've just received the same problem.
`Error in `[<-`(`*tmp*`, unassigned_cells, cell_type_num[i], value = GetScore(activation_prob_to_use,  : 
  subscript out of bounds`

**Comment by weiruo16** [comment]
If possible, could you please share your inputs so that we can take a look (input expression data and input signature matrix)?

**Comment by Elena983** [comment]
yes, I take another data, but still have  'Error in current_pri_matrix[i, ] : subscript out of bounds'

**Comment by Elena983** [comment]
[markers.csv](https://github.com/plevritis-lab/CELESTA/files/12777186/markers.csv)
[countmatrix.csv](https://github.com/plevritis-lab/CELESTA/files/12777188/countmatrix.csv)

**Comment by weiruo16** [comment]
> [markers.csv](https://github.com/plevritis-lab/CELESTA/files/12777186/markers.csv) [countmatrix.csv](https://github.com/plevritis-lab/CELESTA/files/12777188/countmatrix.csv)

There are some minor errors in the input signature matrix. I have made the corresponding changes and highlighted them in a excel file attached. You can save it as a .csv file and run it with CELESTA. Please note that the first number in the lineage should represent the round number in the lineage, and the second number should represent the cell type number in the previous lineage. I managed running your data using the default parameters, which may need some tuning with respect to your images. I attached some results for your reference. 

[markers_modified.xlsx](https://github.com/plevritis-lab/CELESTA/files/12785106/markers_modified.xlsx)
![round1_Cells](https://github.com/plevritis-lab/CELESTA/assets/56096805/46364ab6-b331-427d-9f89-ab89b72596ee)

![round2_Cells](https://github.com/plevritis-lab/CELESTA/assets/56096805/d0e813ce-2ad7-4388-8103-306d168f3742)

**Comment by Elena983** [comment]
Looks great
Many thanks!
